ui/packages/consul-ui/tests/integration/services/repository/auth-method-test.js

```diff
@@ -38,7 +43,9 @@ const undefinedNspace = 'default';
               return Object.assign({}, item, {
                 Datacenter: dc,
                 Namespace: item.Namespace || undefinedNspace,
-                uid: `["${item.Namespace || undefinedNspace}","${dc}","${item.Name}"]`,
+                Partition: item.Partition || undefinedPartition,
+                uid: `["${item.Partition || undefinedPartition}","${item.Namespace ||
+                  undefinedNspace}","${dc}","${item.Name}"]`,
```

Comment:
Similar to serializer tests, repo test changes just add `partition` again and again and again for every single repository.